docs/source/message_requests.md

```diff
@@ -27,4 +27,14 @@ Lost messages may delay catchup for a long time. The node message requests mecha
 - A node requests a ConsistencyProof with median rate of ledger size on other nodes.
 - With the first message request for a ConsistencyProof the node is scheduling a new message request in config.ConsistencyProofTimeout for case the answer to the first request is not received.
 
-If the node has the quorum(n - f - 1) of LedgerStatuses or the quorum(f + 1) of ConsistencyProofs, scheduled request will be canceled.
\ No newline at end of file
+If the node has the quorum(n - f - 1) of LedgerStatuses or the quorum(f + 1) of ConsistencyProofs, scheduled request will be canceled.
+
+###ViewChange messages
+
+Loose a NewView message from primary. It is necessary to finish a view change. A node requests NewView messages from all nodes in NEW_VIEW_TIMEOUT after a view change started. 
+- If an answer is received from a primary is received the node uses them and finishes the view change. 
+- Else it uses a quorum (f+1) of NewView responses from other nodes, finishes the view change and starts catchup.
```

Comment:
I'd prefer to use "otherwise" instead of "else", but this probably is subjective...